disttask/framework/dispatcher/dispatcher.go

```diff
@@ -138,7 +142,10 @@ func (d *dispatcher) Stop() {
 }
 
 // DispatchTaskLoop dispatches the global tasks.
-func (d *dispatcher) DispatchTaskLoop() {
+func (d *dispatcher) DispatchTaskLoop(r interface{}) {
+	if r != nil {
+		logutil.BgLogger().Warn("dispatch task loop run into panic, restart it", zap.String("error", errors.Errorf("%v", r).Error()))
```

Comment:
Does it print the panic stack correctly?

Reply:
I think We can add panic stack into RunWithRecover. It can avoid printing excessive stack trace information.

Reply:
> I think We can add panic stack into RunWithRecover. It can avoid printing excessive stack trace information.

No, I will log it.